client/components/ProgressBar.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { Link } from 'react-router-dom'
+import {
+  Step,
+  Stepper,
+  StepLabel,
+  StepContent,
+} from 'material-ui/Stepper';
+import RaisedButton from 'material-ui/RaisedButton';
+import FlatButton from 'material-ui/FlatButton';
+
+
+
+
+class ProgressBar extends React.Component {
+
+  state = {
+    finished: false,
+    placeIndex: 0,
+  }
+
+  createPlacesArr() {
+    let placesArr = [];
+    for (let i = 0; i <= 4; i++) {
+      placesArr.push(i);
+    }
+    return placesArr;
+  }
+
+  handleNext = () => {
+    const { placeIndex } = this.state;
+    this.setState({
+      placeIndex: placeIndex + 1,
+      finished: placeIndex >= 4 
+    })
+  }
+
+
+
+
+  render() {
+
+    const { finished, placeIndex } = this.state
+    const places = this.createPlacesArr()
+
+    console.log('places', places)
+
+
+    return (
+      <div style={{ maxWidth: 380, maxHeight: 400, margin: 'auto' }}>
+
+
+        <Stepper activeStep={placeIndex} orientation="vertical">
+          {
+            places && places.map(place => (
+              <Step key={place}>
+                <StepLabel>Place's title</StepLabel>
```

Comment:
For `Place's title` we should make this a variable that we can replace later for example:

<StepLabel> {place.title} </StepLabel> 
WHERE 
isPlaceCheckedIn ? place.title = place.title : place.title='??????' 

We wanted to hide the name of the place if they haven't been there yet